mama/c_cpp/src/c/mama/msg.h

```diff
@@ -62,7 +64,8 @@ typedef enum mamaPayloadType_
     MAMA_PAYLOAD_EXEGY      = 'X',
 
 /* Unknown Payload */
-    MAMA_PAYLOAD_UNKNOWN    = 'U'
+    MAMA_PAYLOAD_UNKNOWN    = 'U',
+    MAMA_PAYLOAD_NULL       = '\0'
```

Comment:
Doesn't look like MAMA_PAYLOAD_NULL is used anywhere. Is the purpose of this just to reserve the 0x00 value so that nobody reserves it? If so, I think it should look like MAMA_PAYLOAD_NULL = MAMA_PAYLOAD_ID_NULL where MAMA_PAYLOAD_ID_NULL is #defined elsewhere. That way NULL checks for the payload ID can be performed without depending on the enumerator to tell you how to check for NULL... or just take it out entirely and assume NULL means payload type isn't defined.